dev/bots/run_command.dart

```diff
@@ -147,3 +170,18 @@ Future<void> runCommand(String executable, List<String> arguments, {
     exit(1);
   }
 }
+
+T identity<T>(T x) => x;
+
+/// Flattens a nested list of UTF-8 code units into a single string.
+String flattenToString(List<List<int>> chunks) =>
```

Comment:
This is only used in this file so make it private

Reply:
Fixed.